en/docs/install-and-setup/setup/kubernetes-operators/k8s-api-operator/manage-integration/integration-deployments.md

```diff
@@ -450,4 +481,170 @@ Use the following methods to invoke the inbound endpoints in HTTP and HTTPS tran
 
     ```bash
     curl --cacert <CERT_FILE> https://<HOST-NAME>/<INTEGRATION-NAME>-inbound/<PORT>/<CONTEXT>
-    ```    
\ No newline at end of file
+    ```    
+
+
+## Managing resources for pods in EI deployment
```

Comment:
```suggestion
## Manage resources for pods in a deployment
```

Reply:
I suggest **Integration** instead of EI?

Reply:
updated to `Manage resources for pods of integration deployment`